### ISSUE TITLE
Android strongSwan Profile missed

### ISSUE THREAD

**Issue by flashfoxter** [trailofbits/algo#1305]
### OS / Environment

ubuntu 18.04

### Summary of the problem

Algo VPN does not generate *.sswan file in /configs folder

### Steps to reproduce the behavior

install algo on ubuntu on my vps (ovh)
install instructions: https://github.com/trailofbits/algo

### The way of deployment (cloud or local)

local (vps)

### Expected behavior

here we can see list of files url: [(https://github.com/trailofbits/algo)] 
in Other Devices part ..

Depending on the platform, you may need one or multiple of the following files.
user.sswan: Android strongSwan Profile

but where is no this file user.sswan (or any *.sswan) in /configs folder

### Actual behavior

root@vps639380:~/algo-master/configs/147.135.210.149# ls -l
total 88
-rw------- 1 root root  737 Jan 22 17:57 cacert.pem
-rw-r--r-- 1 root root  297 Jan 22 17:57 config.yml
-rw------- 1 root root 8266 Jan 22 17:57 dan.mobileconfig
-rw------- 1 root root 1153 Jan 22 17:57 dan.p12
lrwxrwxrwx 1 root root   19 Jan 22 17:57 dan.ssh.pem -> pki/private/dan.key
-rw------- 1 root root  530 Jan 22 17:57 ipsec_dan.conf
-rw------- 1 root root   32 Jan 22 17:57 ipsec_dan.secrets
-rw------- 1 root root  531 Jan 22 17:57 ipsec_jack.conf
-rw------- 1 root root   33 Jan 22 17:57 ipsec_jack.secrets
-rw------- 1 root root 8269 Jan 22 17:57 jack.mobileconfig
-rw------- 1 root root 1155 Jan 22 17:57 jack.p12
lrwxrwxrwx 1 root root   20 Jan 22 17:57 jack.ssh.pem -> pki/private/jack.key
drwx------ 9 root root 4096 Jan 22 17:57 pki
-rw------- 1 root root 8427 Jan 22 17:57 windows_dan.ps1
-rw------- 1 root root 8428 Jan 22 17:57 windows_jack.ps1
drwxr-xr-x 4 root root 4096 Jan 22 17:56 wireguard

**Comment by flashfoxter** [comment]
Maybe you can give me the way to generate file by myself ? can't find file from previous versions with this template. I want to use my own vpn client, i don't want to use Wireguard. Thanks.

**Comment by jackivanov** [comment]
@flashfoxter You can take a look at https://wiki.strongswan.org/projects/strongswan/wiki/AndroidVPNClientProfiles

**Comment by flashfoxter** [comment]
Thanks i am already done this, but i am not understand very well  about what "MIIN.." means in p12
{
    "uuid": "a061d140-d3f9-4db7-b2f8-32d6703f4618",
    "name": "Test Profile Certificate",
    "type": "ikev2-cert",
    "remote": {
        "addr": "10.0.2.2" 
    },
    "local": {
        "p12": "MIIN..." 
    }
}

may be you can provide your template generator file, can't find it in commits history. Thanks.

**Comment by jackivanov** [comment]
You are welcome to get help on [Gitter Algo Chat](https://gitter.im/trailofbits/algo)

**Comment by hackslashX** [comment]
> Expected behaviour. It's deprecated. Use WireGuard, please. We'll fix README

On enterprise networks (such as eduroam WiFi in universities), UDP is blocked thus Wireguard doesn't works (unless you setup TCP proxy but that deteriorates connection speed). An option to build strongSwan profiles would be awesome.

**Comment by davidemyers** [comment]
@hackslashX I'm not familiar with eduroam, but according to their [policies](https://www.eduroam.org/wp-content/uploads/2016/05/GN3-12-192_eduroam-policy-service-definition_ver28_26072012.pdf) UDP port 1194 is required to be unblocked for use by OpenVPN. You can have your AlgoVPN use this port for WireGuard by changing `wireguard_port` in `config.cfg` before you deploy.

**Comment by hailfinger** [comment]
Will you accept a patch reverting the Android strongswan support removal part of commit 3488e660ad3535332f59ba5c613d0e1b558e630b?

I'd rather not use out-of-tree kernel patches like Wireguard.
